### ISSUE TITLE
Update required Pelican version to first one supporting the new plugin system

### ISSUE THREAD

**Issue by davidag** [pelican-plugins/webring#2]
I'm trying the new plugin system based on the namespace `pelican.plugins` and I've realized that the current configuration will not allow this plugin to be installed when the next Pelican version is released, which is expected to contain the aforementioned new plugin system.

Currently, `pyproject.toml` contains this dependency:

    pelican = "^4.2"

which implies: `pelican<5.0,>=4.2`.

Unless I'm missing something, a new version of this plugin should be released with the updated dependency before the release of the next Pelican version.

**Comment by justinmayer** [comment]
Thank you for bringing this up, David. I'm increasingly leaning toward using the 4.5 moniker for the next Pelican release, for the simple reason that 5.0 seems too grand and 4.3 seems too limited. Other dropping Python 2.7 (and maybe 3.5) support, there shouldn't be significant backward-incompatible aspects to current Pelican master, which is why I feel that perhaps marking the next release as version 4.5 would be a nice middle-ground.

Nonetheless, your original point is still relevant, and I think the corresponding line should indeed be changed to `pelican = "^4.5"` once that version is released.

**Comment by davidag** [comment]
Using 4.5 seems perfectly reasonable to me and I'll update this issue accordingly. Thanks!

**Comment by justinmayer** [comment]
Hi @davidag. I just released Pelican 4.5. The currently-released version of Webring will work just fine without updating the required Pelican version, as I just tested by running `pip install pelican-webring` in a fresh virtual environment, since doing so will automatically install Pelican 4.5.

The only thing that *won't* work as expected is if someone tries to `pip install pelican-webring` into a Pelican 4.2 (or lower) environment. I don't know how likely that is. On the other hand, I can't think of any reason *not* to require Pelican 4.5+. In summary, I think it's up to you to decide whether issuing a new plugin release is necessary. 😊
